src/widget/wtrackmenu.cpp

```diff
@@ -185,6 +187,38 @@ void WTrackMenu::createMenus() {
                 });
     }
 
+    if (featureIsEnabled(Feature::FindOnWeb)) {
+        DEBUG_ASSERT(!m_pFindOnWebMenu);
+        m_pFindOnWebMenu = make_parented<WFindOnWebMenu>(this); //¿Is it okay to have parent menu?
```

Comment:
Yes

Reply:
Perfect, thank you! I will delete the comment lines before we propose it to public.